### PR TITLE
Formulary

### PR DESCRIPTION
**Pull request by rubenaltair**
Developed the formulary (component/contact)

### CODE REVIEW

File: src/components/Contact.vue [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+      <div class="field advise">
+         <p>Si desea adjuntar uno o más archivos pulse en "Verificar y enviar", 
+          seleccione "Insertar" en la parte superior y elija "archivos"</p>
+      </div>
+      
+      <div class="click">
+        <button class="btn reset" @click="reset">Resetear</button>
+        <a :href="mailto">
+          <button class="btn send">Verificar y enviar</button>
```
Comment (nestorhdez) [inline src/components/Contact.vue]: The tag button isn't needed.

File: src/components/Contact.vue [hunk 2]
```diff
@@ -0,0 +1,120 @@
+<template>
+  <div class="formulary">
+    
+      <div class="field">
+        <h2>Nombre</h2>
```
Comment (nestorhdez) [inline src/components/Contact.vue]: Use labels insted h2 for better semantic

Comment (rubenaltair) [reply src/components/Contact.vue]: do it

**Comment by rubenaltair** [comment]
all oki doki

**Pull request closed without merging**